addons/mail/static/tests/qunit_suite_tests/components/notification_list_notification_group_tests.js

```diff
@@ -83,45 +83,45 @@ QUnit.module("mail", {}, function () {
             );
         });
 
-        QUnit.skipRefactoring("mark as read", async function (assert) {
+        QUnit.test("mark as read", async function (assert) {
             assert.expect(2);
 
             const pyEnv = await startServer();
-            const mailChannelId1 = pyEnv["mail.channel"].create({});
             const mailMessageId1 = pyEnv["mail.message"].create({
                 message_type: "email", // message must be email (goal of the test)
-                model: "mail.channel", // expected value to link message to channel
-                res_id: mailChannelId1,
                 res_model_name: "Channel", // random res model name, will be asserted in the test
             });
+            pyEnv["mail.channel"].create({
+                message_ids: [mailMessageId1],
+                channel_member_ids: [
+                    [0, 0, { partner_id: pyEnv.currentPartnerId, seen_message_id: mailMessageId1 }],
+                ],
+            });
             // failure that is expected to be used in the test
             pyEnv["mail.notification"].create({
                 mail_message_id: mailMessageId1, // id of the related message
                 notification_status: "exception", // necessary value to have a failure
                 notification_type: "email",
             });
-            const { click } = await start();
+            await start();
             await click(".o_menu_systray .dropdown-toggle:has(i[aria-label='Messages'])");
             assert.containsOnce(
                 document.body,
-                ".o_NotificationGroupView_markAsRead",
+                ".o-mail-notification-item i[title='Mark As Read']",
                 "should have 1 mark as read button"
             );
 
-            await afterNextRender(() => {
-                document.querySelector(".o_NotificationGroupView_markAsRead").click();
-            });
-            assert.containsNone(
+            await click(".o-mail-notification-item i[title='Mark As Read']");
+            assert.containsOnce(
                 document.body,
-                ".o_NotificationGroupView",
+                ".o-mail-notification-item",
                 "should have no notification group"
             );
         });
```

Comment:
assertion comment does not match assertion. And it leaks showing of channel preview.
Better make a more specific target for assertion, and have it has `None`.